models/staging/jaffle_shop/stg_total_customer_orders.sql

```diff
@@ -0,0 +1,5 @@
+select
+    fct_orders.customer_id
+    ,sum(fct_orders.amount) as lifetime_value

```

Comment:
no aggregation in staging layer -- staging models should just be renaming and recasting any basic transformations for sources --- this is too specific to be a staging model